builder-22/builder.toml

```diff
@@ -83,12 +87,17 @@ version = "0.16.1"
 [[order]]
   [[order.group]]
     id = "heroku/java-function"
-    version = "0.3.43"
+    version = "0.3.44"
 
 [[order]]
   [[order.group]]
     id = "heroku/java"
-    version = "0.6.9"
+    version = "0.6.10"
+
+[[order]]
+  [[order.group]]
+    id = "heroku/scala"
```

Comment:
Does that mean the Scala getting started exclusion in the CI workflow can be removed now?
https://github.com/heroku/builder/blob/f5269bd3754f94cfae582fba0c7d6c31407b8da0/.github/workflows/build-test-publish.yml#L64-L65

Reply:
Yes! I removed it just now in [d4deb23](https://github.com/heroku/builder/pull/341/commits/d4deb23b676d1f733b7546aeaf3ee33a00af27af)